### PR TITLE
Update docs for identity

### PR DESCRIPTION
**Pull request by TheIronDev**
Update docs for identity (resolving warnings)

### CODE REVIEW

File: src/v2/providers/identity.ts [hunk 1]
```diff
@@ -20,17 +20,15 @@ interface InternalOptions {
...
 export interface BlockingOptions extends options.GlobalOptions {
   idToken?: boolean;
   accessToken?: boolean;
   refreshToken?: boolean;
 }
 
-/**
- * Handle an event that is triggered before a user is created.
- */
+/** @beta Handle an event that is triggered before a user is created. */
```
Comment (colerogers) [inline src/v2/providers/identity.ts:31]: we should yank beta per our offline discussion

Comment (TheIronDev) [reply src/v2/providers/identity.ts]: Done!

File: src/v2/providers/identity.ts [hunk 2]
```diff
@@ -40,6 +41,12 @@ export function beforeUserCreated(
     | void
     | Promise<void>
 ): BlockingFunction;
+
+/**
+ * Handles an event that is triggered before a user is created.
+ * @param opts - Object containing Function options
```
Comment (egilmorez) [inline src/v2/providers/identity.ts:47]: Is Function a literal?  Just "function" could work.

File: src/v2/providers/identity.ts [hunk 3]
```diff
@@ -50,6 +57,12 @@ export function beforeUserCreated(
     | void
     | Promise<void>
 ): BlockingFunction;
+
+/**
+ * Handles an event that is triggered before a user is created
+ * @param optsOrHandler - Either an object containing Function options, or an event handler (run before user creation)
+ * @param handler? - If defined, an wvent handler which is run every time before a user is created
```
Comment (egilmorez) [inline src/v2/providers/identity.ts:64]: Typo, "event"

Also, should "Function" be capped?  Please check throughout, thanks!

**Review by egilmorez — COMMENTED** [review]
A couple of things to look at, thanks!

File: src/v2/providers/identity.ts (hunk 3 above)
Comment (TheIronDev) [reply src/v2/providers/identity.ts:64]: Updated, I think it should be lowercased, updated throughout

**Comment by TheIronDev** [comment]
> A couple of things to look at, thanks!

Thank you!!

**Review by egilmorez — APPROVED** [review]
LG, thanks!

**Pull request merged**